loading/lazyload/original-base-url-applied-2-tentative.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<head>
+  <title>Deferred images with loading='lazy' use the original
+         base URL specified at the parse time</title>
+  <link rel="author" title="Rob Buis" href="mailto:rbuis@igalia.com">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <script src="common.js"></script>
+  <base href='/loading/lazyload/resources/'>
+</head>
+
+<!--
+Marked as tentative until https://github.com/whatwg/html/pull/3752 is landed.
+-->
+
+<script>
+  let has_window_loaded = false;
+
+  async_test(function(t) {
+    const below_viewport_img = new ElementLoadPromise("below_viewport_img");
```

Comment:
This variable cannot be in the `async_test` function's scope. If it is, then when you try and reference the variable from the `onload` and `onerror`, it will not exist (instead is undefined).

Reply:
Done in crrev